src/openforms/registrations/contrib/objects_api/tests/test_objecttypes_client.py

```diff
@@ -0,0 +1,67 @@
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from zgw_consumers.constants import APITypes, AuthTypes
+from zgw_consumers.models import Service
+
+from openforms.utils.tests.vcr import OFVCRMixin
+
+from ..client import get_objecttypes_client
+from ..models import ObjectsAPIConfig
+
+OBJECTTYPES_API_BASE_URL = os.getenv(
+    "OBJECTTYPES_API_BASE_URL", "http://localhost:8001/api/v2/"
+)
```

Comment:
Managed through docker-compose with a fixed URL, so doesn't need to be configurable through the env.